syntax = "proto3";

message MsgResponse{
    string msg = 1;
}